com.unity.ml-agents/Runtime/Policies/BehaviorParameters.cs

```diff
@@ -121,11 +119,12 @@ public string behaviorName
 
         /// <summary>
         /// Whether or not to use all the sensor components attached to child GameObjects of the agent.
+        /// Note that changing this after the Agent has been initialized will not have any effect.
```

Comment:
Will it have an effect when the agent changes model for example?

Reply:
No, it's only used when initializing: https://github.com/Unity-Technologies/ml-agents/blob/463b90b98cc84cec476e7d1f31d06da4297335dc/com.unity.ml-agents/Runtime/Agent.cs#L535-L542